extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -32,55 +35,105 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
     /** @private {?Element} */
     this.element_ = element;
 
-    /** @private {?Element} */
-    this.image_ = null;
+    /** @private {?Element} The element that is transitioned. */
+    this.ampImgEl_ = dev().assertElement(
+      this.element_.querySelector('amp-img')
+    );
 
-    /** @private {?string} */
-    this.x_ = null;
+    /** @public {?string} Sent to siblings to update their position. */
+    this.x = this.element_.getAttribute('x') || '0%';
```

Comment:
Where is it that this property gets used outside this class? Unless I'm missing sth, it could be kept private right?

Reply:
As per our discussion, these are passed to a method as arguments so they can be private as you suggest.